ui/v2/src/components/Movies/MovieDetails/Movie.tsx

```diff
@@ -0,0 +1,205 @@
+interface IProps extends IBaseProps {}
+
+export const Movie: FunctionComponent<IProps> = (props: IProps) => {
+  const isNew = props.match.params.id === "new";
+
+  // Editing state
+  const [isEditing, setIsEditing] = useState<boolean>(isNew);
+
+  // Editing movie state
+  const [front_image, setFront_Image] = useState<string | undefined>(undefined);
+  const [back_image, setBack_Image] = useState<string | undefined>(undefined);
+  const [name, setName] = useState<string | undefined>(undefined);
+  const [aliases, setAliases] = useState<string | undefined>(undefined);
+  const [duration_movie, setDuration_movie] = useState<string | undefined>(undefined);
+  const [date_movie, setDate_movie] = useState<string | undefined>(undefined);
+  const [rating_movie, setRating] = useState<number | undefined>(undefined);
+  const [director, setDirector] = useState<string | undefined>(undefined);
+  const [synopsis, setSynopsis] = useState<string | undefined>(undefined);
+  const [url, setUrl] = useState<string | undefined>(undefined);
+
+  // Movie state
+  const [movie, setMovie] = useState<Partial<GQL.MovieDataFragment>>({});
+  const [imagePreview, setImagePreview] = useState<string | undefined>(undefined);
+  const [backimagePreview, setBackImagePreview] = useState<string | undefined>(undefined);
+
+  // Network state
+  const [isLoading, setIsLoading] = useState(false);
+
+  const { data, error, loading } = StashService.useFindMovie(props.match.params.id);
+  const updateMovie = StashService.useMovieUpdate(getMovieInput() as GQL.MovieUpdateInput);
+  const createMovie = StashService.useMovieCreate(getMovieInput() as GQL.MovieCreateInput);
+  const deleteMovie = StashService.useMovieDestroy(getMovieInput() as GQL.MovieDestroyInput);
+
+  function updateMovieEditState(state: Partial<GQL.MovieDataFragment>) {
+    setName(state.name);
+    setAliases(state.aliases);
+    setDuration_movie(state.duration_movie);
+    setDate_movie(state.date_movie);
+    setRating(state.rating_movie == null ? NaN : state.rating_movie);
+    setDirector(state.director);
+    setSynopsis(state.synopsis);
+    setUrl(state.url);
+  }
+
+  useEffect(() => {
+    setIsLoading(loading);
+    if (!data || !data.findMovie || !!error) { return; }
+    setMovie(data.findMovie);
+  }, [data, loading, error]);
+
+  useEffect(() => {
+    setImagePreview(movie.front_image_path);
+    setBackImagePreview(movie.back_image_path);
+    setFront_Image(undefined);
+    setBack_Image(undefined);
+    updateMovieEditState(movie);
+    if (!isNew) {
+      setIsEditing(false);
+    }
+  }, [movie, isNew]);
+
+  function onImageLoad(this: FileReader) {
+    setImagePreview(this.result as string);
+    setFront_Image(this.result as string);
+    
+  }
+
+  function onBackImageLoad(this: FileReader) {
+    setBackImagePreview(this.result as string);
+    setBack_Image(this.result as string);
+  }
+
+
+  ImageUtils.addPasteImageHook(onImageLoad);
+  ImageUtils.addPasteImageHook(onBackImageLoad);
```

Comment:
Well, I tried to make this change but i saw a problem. When I choosed one image the first time, the frontImage was loaded. When I choosed one image the second time then the backimage was loaded. Well, I tried to load a new image but now "front_image is different to undefined" and then changed the backimage. I was not be able to change the frontimage.
I'd rather let the load the images with two buttons: "one to frontimage and one to backimage" . I think is better.

Reply:
Yeah, that's the compromise you would get. Not sure there is a much better way to handle pasting. The alternative would be to just remove the pasting altogether for the movies form since it doesn't seem very useful to have one image pasted in both front and back.